### PR TITLE
replace img link

### PR DESCRIPTION
**Pull request by LarryMatte**
screen shot 2015-01-21 at 1 46 10 pm

### CODE REVIEW

File: site/src/main/markdown/gwtp/contributing/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ TODO
 #Building
 
 ##Builds
-[![Build Status](https://buildhive.cloudbees.com/job/ArcBees/job/GWTP/badge/icon)](https://buildhive.cloudbees.com/job/ArcBees/job/GWTP/)
+[![Build Status](https://travis-ci.org/ArcBees/GWTP.svg?branch=master)](https://travis-ci.org/ArcBees/GWTP)
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/contributing/index.md]: In fact I think the link should be `[Build Status/statusIcon)](http://teamcity.arcbees.com/project.html?projectId=GwtPlatformGwtp)`

File: site/src/main/markdown/gwtp/contributing/index.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ TODO
...
 
 ##Build Server
 * [TeamCity Build Server](http://teamcity.gonevertical.org) - Snapshots and releases building here
```
Comment (meriouma) [inline site/src/main/markdown/gwtp/contributing/index.md]: And here it should be `http://teamcity.arcbees.com/`. You could also remove the 2 next lines at the same time :P

File: site/src/main/markdown/gwtp/contributing/index.md (hunk 1 above)
Comment (LarryMatte) [inline site/src/main/markdown/gwtp/contributing/index.md]: @meriouma 
well that link doesn't work
screen shot 2015-01-21 at 2 03 27 pm

Comment (meriouma) [inline site/src/main/markdown/gwtp/contributing/index.md]: Weird, works on github. Pegdown probably doesn't like the () inside the link. Leave it like that for now, I'll check it out

**Comment by spg** [comment]
LGTM

**Pull request merged**